GUIDE.md

```diff
@@ -0,0 +1,159 @@
+it can quickly become confusing and hard to maintain. Documents that start to
+transform in this way should be **split up**. When a document about a topic
+starts to describe two ideas, the core concept of both should become a
+directory that documents for both ideas are put into. For example:
+
+    topics/streams.md ->
+      topics/streams/using-a-stream.md      # for users of streams
+      topics/streams/writing-a-stream.md    # for authors of streams
+      topics/streams/index.md               # links to both of the above
+
+In other cases, reference documentation may have need of describing a topic in
+order to explain an API. In that case, the topic should be described
+separately, as a document in `topics/`, and linked to from the reference in
+`references/`. This is especially true when multiple references need to
+describe the same topic!
+
+We believe good documentation is fractal. Guides, topics, and reference may all
+repeat that organization internally, so long as it does not interfere with the
+communication of the core idea of the document. For example, it's good to add
+links to reference materials at the end of topic docs, but it would be unwise
+to enumerate a modules entire list of methods inside that same doc. Another
+example: it's great to include *brief* example code in the reference
+documentation, but it is not so great to include a full program there.
+
+To summarize:
+
+* Structure is of paramount importance.
+* Decide on an **audience** and an **idea** you want to convey to that
+audience.
+* If the reader is to follow a process to get to a goal, it's a **guide.**
+* If what you're writing conveys a concept to help the reader in decision
+making, it's a **topic.**
+* When the reader is consulting your document to find a specific piece of
+information, it's **reference.**
+* Each document should clearly convey *one* idea to *one* audience. All
+strategies are in service of that goal.
+
+## How We Write
+
+* Documents are written in markdown files. 
+* Those files should be written in **`lowercase-with-dashes.md`.**
+  * Underscores in filenames are allowed only when they are present in the
+    topic the document will describe (e.g., `child_process`.)
+  * Filenames should be **lowercase**.
+  * Older files may use the `.markdown` extension. These may be ported to `.md`
+    at will. **Prefer `.md` for all new documents.**
+* Documents should be word-wrapped at 80 characters.
+* Documents should not contain trailing whitespace.
+* Mechanical issues, like spelling and grammar, should be identified by tools,
+  insofar as is possible. If not caught by a tool, they should be pointed out by
+  human reviewers.
+* US spelling is preferred. "Capitalize" vs. "Capitalise", "color" vs.
+  "colour", etc.
+* Though controversial, the [Oxford comma][] is preferred for clarity's sake.
+* Generally avoid personal pronouns in reference documentation ("I", "you",
```

Comment:
We mention personal pronouns, did we want to make an explicit point regarding using non-gender specific pronounds?